src/Interpreters/DatabaseCatalog.cpp

```diff
@@ -155,7 +162,12 @@ void DatabaseCatalog::shutdownImpl()
     tables_marked_dropped.clear();
 
     std::lock_guard lock(databases_mutex);
-    assert(std::find_if_not(uuid_map.begin(), uuid_map.end(), [](const auto & elem) { return elem.map.empty(); }) == uuid_map.end());
+    assert(std::find_if(uuid_map.begin(), uuid_map.end(), [](const auto & elem)
```

Comment:
nested lambda with const auto parameters...
`mapping.second.second` -- too complicated to read. 
Are we just checking, that our uuid_map contains no active tables?